react-native-gutenberg-bridge/index.js

```diff
@@ -14,8 +14,11 @@ export const mediaSources = {
 	deviceLibrary: 'DEVICE_MEDIA_LIBRARY',
 	deviceCamera: 'DEVICE_CAMERA',
 	siteMediaLibrary: 'SITE_MEDIA_LIBRARY',
+	mediaEditor: 'MEDIA_EDITOR',
 };
 
+export const showMediaEditorButton = isIOS;
```

Comment:
> Since the feature is not ready yet, it might be good to add also a __DEV__ check too.

Added in 5ca3c27a04d57c375ed565cc4ab971fbed44beb8

> It would be also great if we ask this to the parent app, so we avoid branching platforms and WPAndroid would say false directly. But this is not really necessary at this stage. Just an idea 👍

I thought about doing that! I gave up because:

* This will mean more code (in the bridge and in the apps)
* Once we have the feature on both platforms, we will remove all this conditional code. Less code to remove = better.

Does that sound reasonable to you?

Reply:
Works for me 👍 

> Once we have the feature on both platforms, we will remove all this conditional code. Less code to remove = better.

Is good to think about Gutenberg as a library for others to implement too, and other library users might not want to have that edit button there. In the other hand, right now 3rd parties are not priority for the mobile version, so let's keep it as it is.